doc/pages/architecture/080-mqtt-v1-protocol.md

```diff
@@ -0,0 +1,139 @@
+# Astarte MQTT v1 Protocol
+
+Astarte MQTT v1 Protocol allows communication between devices and Astarte. It is the first protocol that has been implemented in Astarte, and it has been designed with Astarte requirements in mind.
+
+One of MQTT key points is that it doesn't mandate the data serialization format, so any application might implement its own format. Data serialization might be a tricky task and protocols might be hard to design, Astarte takes care of this burden and offers a ready to use protocol.
```

Comment:
> Astarte takes care of this burden and offers a ready to use protocol.

*Astarte MQTT* takes care of this and provides a higher level protocol which abstracts this detail from the end user.